src/samplers/multijitter.cpp

```diff
@@ -0,0 +1,159 @@
+#include <mitsuba/core/profiler.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/render/sampler.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**!
+
+.. _sampler-multijitter:
+
+Correlated Multi-Jittered sampler (:monosp:`multijitter`)
+---------------------------------------------------------
+
+.. pluginparameters::
+
+ * - sample_count
+   - |int|
+   - Number of samples per pixel. This value might be slightly increased by the plugin to produce a
```

Comment:
`This value might be slightly ` -> `The sampler may internally choose to slightly increase [..] to create a subdivision into strata that has an aspect ratio close to one`